build.ps1

```diff
@@ -0,0 +1,183 @@
+#!/usr/bin/env pwsh
+#
+# Copyright (c) Microsoft. All rights reserved.
+# Licensed under the MIT license. See LICENSE file in the project root for full license information.
+#
+
+[CmdletBinding()]
+param(
+    [switch]
+    $Clean,
+
+    [switch]
+    $Bootstrap,
+
+    [switch]
+    $Test,
+
+    [switch]
+    $NoBuild,
+
+    [switch]
+    $Deploy,
+
+    [string]
+    $CoreToolsDir,
+
+    [string]
+    $Configuration = "Debug",
+
+    [string]
+    $BuildNumber = '0'
+)
+
+#Requires -Version 6.2
+
+$PowerShellVersion = '7.2'
+$TargetFramework = 'net6.0'
+$ModuleName = 'AzureFunctions.PowerShell.SDK'
+$RepoName = 'azure-functions-powershell-library'
+
+function Get-FunctionsCoreToolsDir {
+    if ($CoreToolsDir) {
+        $CoreToolsDir
+    } else {
+        $funcPath = (Get-Command func).Source
+        if (-not $funcPath) {
+            throw 'Cannot find "func" command. Please install Azure Functions Core Tools: ' +
+                  'see https://github.com/Azure/azure-functions-core-tools#installing for instructions'
+        }
+
+        # func may be just a symbolic link, so we need to follow it until we find the true location
+        while ((((Get-Item $funcPath).Attributes) -band 'ReparsePoint') -ne 0) {
+            $funcPath = (Get-Item $funcPath).Target
+        }
+
+        $funcParentDir = Split-Path -Path $funcPath -Parent
+
+        if (-not (Test-Path -Path $funcParentDir/workers/powershell -PathType Container)) {
+            throw 'Cannot find Azure Function Core Tools installation directory. ' +
+                  'Please provide the path in the CoreToolsDir parameter.'
+        }
+
+        $funcParentDir
+    }
+}
+
+function Deploy-FunctionsSDKModule {
+    $ErrorActionPreference = 'Stop'
+
+    $publishDir = "./$RepoName/bin/$Configuration/$TargetFramework/publish/*" 
+
+    $powerShellWorkerModuleDir = "$(Get-FunctionsCoreToolsDir)/workers/powershell/$PowerShellVersion/Modules/$ModuleName"
+
+    Write-Log "Deploying module to $powerShellWorkerModuleDir..."
+
+    if (-not $IsWindows) {
+        sudo chmod -R a+w $powerShellWorkerModuleDir
+    }
+
+    if (!(Test-Path $powerShellWorkerModuleDir)) {
+        New-Item $powerShellWorkerModuleDir -ItemType directory
+    }
+
+    Remove-Item -Path $powerShellWorkerModuleDir/* -Recurse -Force
+    Copy-Item -Path $publishDir -Destination $powerShellWorkerModuleDir -Recurse -Force
+
+    Write-Log "Deployed module to $powerShellWorkerModuleDir"
+}
+
+Import-Module "$PSScriptRoot/tools/helper.psm1" -Force
+
+# Bootstrap step
+if ($Bootstrap.IsPresent) {
+    Write-Log "Validate and install missing prerequisits for building ..."
```

Comment:
```suggestion
    Write-Log "Validate and install missing prerequisites for building ..."
```

Reply:
Fixed here, but this is probably also in the worker build script (?)